charts/consul/test/unit/client-daemonset.bats

```diff
@@ -1838,7 +2235,7 @@ rollingUpdate:
 
   local actual=$(echo $object |
     yq -r '.volumes[] | select(.name == "consul-ca-cert") | length > 0' | tee /dev/stderr)
-  [ "${actual}" = "" ]
+  [ "${actual}" = "true" ]
```

Comment:
I think this has to be empty like the original test was asserting because when vault is enabled there should be no ca-cert volumes